test/jasmine/tests/gl2d_click_test.js

```diff
@@ -444,3 +444,149 @@ describe('Test hover and click interactions', function() {
         .then(done);
     });
 });
+
+describe('Test gl2d lasso/select:', function() {
```

Comment:
Works fine for me:
![image](https://user-images.githubusercontent.com/300067/27198582-c9dc3b58-51e0-11e7-92ef-a5ec14a42ed3.png)
Although couple of times `'should output correct event data for scattergl'` failed due to delay or something.